frontend/src/modules/Environments/views/EnvironmentCreateForm.js

```diff
@@ -298,171 +299,192 @@ const EnvironmentCreateForm = (props) => {
-                              />
-                            </IconButton>
-                          </CopyToClipboard>
-                          {`cdk bootstrap --trust ${trustedAccount} -c @aws-cdk/core:newStyleStackSynthesis=true --cloudformation-execution-policies arn:aws:iam::aws:policy/AdministratorAccess aws://ACCOUNT_ID/REGION`}
-                        </Typography>
-                      </CardContent>
-                    </Card>
-                  </Grid>
-                  <Grid item xs={6}>
-                    <Card>
-                      <CardContent>
-                        <Typography
-                          sx={{ fontSize: 14 }}
-                          color="text.secondary"
-                          gutterBottom
-                        >
-                          If Using Custom CDK Execution Policy (From Step 1):
-                        </Typography>
-                        <Typography color="textPrimary" variant="subtitle2">
-                          <CopyToClipboard
-                            onCopy={() => copyNotification()}
-                            text={`cdk bootstrap --trust ${trustedAccount} -c @aws-cdk/core:newStyleStackSynthesis=true --cloudformation-execution-policies arn:aws:iam::ACCOUNT_ID:policy/DataAllCustomCDKPolicy aws://ACCOUNT_ID/REGION`}
-                          >
-                            <IconButton>
-                              <CopyAllOutlined
-                                sx={{
-                                  color:
-                                    theme.palette.mode === 'dark'
-                                      ? theme.palette.primary.contrastText
-                                      : theme.palette.primary.main
-                                }}
-                              />
-                            </IconButton>
-                          </CopyToClipboard>
-                          {`cdk bootstrap --trust ${trustedAccount} -c @aws-cdk/core:newStyleStackSynthesis=true --cloudformation-execution-policies arn:aws:iam::ACCOUNT_ID:policy/DataAllCustomCDKPolicy aws://ACCOUNT_ID/REGION`}
-                        </Typography>
-                      </CardContent>
-                    </Card>
-                  </Grid>
-                </Grid>
-              </Box>
-              {process.env.REACT_APP_ENABLE_PIVOT_ROLE_AUTO_CREATE ===
-              'True' ? (
+              {config.core.custom_env_linking_text !== undefined ? (
                 <Box>
-                  <Typography color="textSecondary" variant="subtitle2">
-                    3. As part of the environment CloudFormation stack data.all
-                    will create an IAM role (Pivot Role) to manage AWS
-                    operations in the environment AWS Account.
-                  </Typography>
+                  <Typography
+                    color="textSecondary"
+                    variant="subtitle2"
+                    dangerouslySetInnerHTML={{
```

Comment:
We need to sanitize the text before adding to prevent XSS attack. A good way is to use [DOMPurify](https://github.com/cure53/DOMPurify).
Maybe we create a small component wrapper like this:
`<SanitizedText text={config.core.custom_env_linking_text} />`

Reply:
@noah-paige @dlpzx are you happy with this? Do you want me to introduce a new component and pull in a new library DOMPurify?

Reply:
@mkhidir-aws I don't really see the point of sanitizing it because it's coming out of a config? Why would we want to sanitize it?

Reply:
There is any possibility that the config file can be modified by external sources and it contains content that could be changed, I think it's best to sanitize it.